### ISSUE TITLE
Add an option to ignore color strings

### ISSUE THREAD

**Issue by ivan-aksamentov** [edvardchen/eslint-plugin-i18next#18]
Add an option to ignore hex color strings.

For example:

```js
{ ignoreColors: true }

```

This would ignore color strings like:

```js
"#aaa"
"#aaaa"
"#ababab"
"#abababab"
"rgb(123, 123, 123)"
"rgba(123, 123, 123, 123)"
"hsl(...)"

```

These strings are not translatable anyways.

**Comment by edvardchen** [comment]
If you use color in JSX attributes like `styleName` or `style`, that would be ok. 

If it is not enough,  I think we can do more, to ignore the color strings



**Comment by ivan-aksamentov** [comment]
@edvardchen No, `style` and `styleName` is not enough. There might be places where programmer is expected to pass colors.

Sometimes these are simply attibutes:

```jsx
<Component background="#ccc" foreground="#aaa" />
```

but there are many examples of packages (e.g. various styleguides and theming engines) where you have color strings in completely random places, like in arrays, objects, maps etc.:

```jsx
const theme = {
   primary: "#ababab",
   danger: "ffaaaa"
}

<ThemeProvider theme={theme}>
  <App/>
</ThemeProvider>
```

Example: https://styled-components.com/docs/advanced#theming


With these patterns present in the app it is very hard to use this plugin.

If there would be regexes, then this feature could be implemented in userspace: I would just add a regex for hex colors to ignore list.

**Comment by edvardchen** [comment]
@ivan-aksamentov You already can do that.

```js
/*eslint i18next/no-literal-string: ["error", {"ignore": ["^#"]}]*/
const a = '#abc';
```

**Comment by edvardchen** [comment]
try `v4` and you can set `ignore` option with regex
